variants/frontend-base/template.rb

```diff
@@ -8,6 +8,7 @@
 
 run "mv app/javascript/* app/frontend"
 run "rm -rf app/javascript"
+run "rm .browserslistrc"
```

Comment:
Does something create this file? If so, how easy would it be to persuade that thing to not create it?

Reply:
@eoinkelly I suggested to @mischa-s that `rails new` with webpacker probably created it

Reply:
@eoinkelly confirming that `rails g webpacker:install` creates it along with `babel.config.js` and `postcss.config.js`